css-two-column-layout/styles.css

```diff
@@ -0,0 +1,82 @@
+body {
+  font-family: Karla, sans-serif;
+}
+
+h2 {
+  text-align: center;
+  font-size: 3.8rem;
+  letter-spacing: 0.3rem;
+  margin-bottom: 30px;
+}
+
+.container {
+  max-width: 1200px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-half {
+  flex-basis: 50%;
+}
+
+.column-one-fourth {
+  flex-basis: 30%;
+}
+
+.column-full p {
+  text-align: center;
+  font-size: 17px;
+  letter-spacing: 0.03rem;
+  line-height: 30px;
+  margin-bottom: 5rem;
+  margin-left: 12rem;
+  width: 800px;
+  color: #676767;
+}
+
+img {
+  margin-top: -2rem;
+}
+
+.column-one-fourth p {
+  font-size: 18px;
+  color: #676767;
+  letter-spacing: 0.02rem;
+  line-height: 1.8rem;
+  width: 350px;
+  padding-bottom: 5rem;
+}
+
+.column-one-fourth h3 {
+  color: #1c1c1c;
+  width: 250px;
+}
+
+.row .column-full {
+  justify-content: center;
+  width: 800px;
+  align-content: center;
+  flex-wrap: wrap;
+}
```

Comment:
If you feel like you need to make a `column` class into a flexbox, that usually means you just need a child `row` inside that `column`

Reply:
<img width="909" alt="Screen Shot 2022-07-09 at 2 50 46 PM" src="https://user-images.githubusercontent.com/97363006/178123819-b2dcef79-249c-4771-a830-7c49afd15b14.png">